### ISSUE TITLE
[Bug] ArchLinux run error

### ISSUE THREAD

**Issue by zxp19821005** [EntityPlantt/DK-CPP#24]
I've ported it to ArchLinux AUR: https://aur.archlinux.org/packages/dk-c++-bin
When I run the app,I found some bugs:
- `icon.ico` in `app.js` and `installer.js` should be `icon.png`
- `assoc.bat` can't be found in the package.
- Many `.exe` existed in `app.js` and `preload.js`,which can't run under linux.

**Comment by EntityPlantt** [comment]
Yeah, I kinda made it only for Windows. As a Linux user nowadays I'll make an update to be runnable on Linux, too.
Those `exe`s and `bat`s are there for elevation workarounds on Windows. As `sudo` (and `doas`) exist on Linux, it should be much easier. On Windows it still doesn't work if the app has no administrator permissions.
I've never thought about making a Linux version, but I'll make one as soon as I can. I'm busy these weeks, so I really don't know an exact date when I'll create a Linux branch (or possibly merge all into one).

**Comment by zxp19821005** [comment]
Thanks a lot!

**Comment by EntityPlantt** [comment]
@zxp19821005 I pushed some commits on the main branch (https://github.com/EntityPlantt/DK-CPP/compare/87a361d...main), works on my Fedora PC with Gnome. Try running it on Arch. Which terminal are you using? Which other terminals should I implement for Linux?

**Comment by EntityPlantt** [comment]
Here's how it works on Fedora with `gnome-terminal`.

https://github.com/EntityPlantt/DK-CPP/assets/92634082/da992f7c-4f86-4f47-8ce2-ca2916881867
